BookShelfBL/Library.cs

```diff
@@ -0,0 +1,301 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BookShelfBL
+{
+    /// <summary>
+    /// Class containing all methods for data exchange with the database.
+    /// Creates a temporary set of data to work with.
+    /// </summary>
+    public class Library : ILibrary
+    {
+        private List<Book> _books;
+        private List<Author> _authors;
```

Comment:
please in the level4 update approach of using this collection to that variant that we had discussed.